pgdrive/pg_config/pg_blocks.py

```diff
@@ -18,6 +19,8 @@ class PGBlock:
         OutRampOnStraight: 0.1,
         InFork: 0.00,
         OutFork: 0.00,
+        InBottleneck: 0.00,
+        OutBottleneck: 0.00
```

Comment:
What is this??

Reply:
The sampling prob of these blocks is 0 now and it will not be sampled when executing BIG. These blocks are special, and the observation may need some changes.